### DISCUSSION TITLE
FoxyTab needs a cheat sheet on redirect operators

### DISCUSSION THREAD

**Discussion by argarner** [erosman/support#594] [Q&A]
I always have trouble setting up redirects in FoxyTab. Most generic regex operators do not seem to be supported, and I can't find any docs on what is or is not supported.

Please provide a cheat sheet of what can be done, or if a particular flavor of regex is supported, please specify.

**Comment by erosman** [comment]
Wasn't the Redirect in the Help section useful?

> Most generic regex operators do not seem to be supported,

Can you give me an example?
